Makefile.e2e

```diff
@@ -136,14 +137,23 @@ kube-ovn-lb-svc-conformance-e2e:
 	ginkgo $(GINKGO_PARALLEL_OPT) --randomize-all --always-emit-ginkgo-writer \
 		--focus=CNI:Kube-OVN ./test/e2e/lb-svc/lb-svc.test
 
-.PHONY: kube-ovn-eip-conformance-e2e
-kube-ovn-eip-conformance-e2e:
-	ginkgo build $(E2E_BUILD_FLAGS) ./test/e2e/ovn-eip
+.PHONY: iptables-vpc-nat-gw-conformance-e2e
+iptables-vpc-nat-gw-conformance-e2e:
+	ginkgo build $(E2E_BUILD_FLAGS) ./test/e2e/iptables-vpc-nat-gw
 	E2E_BRANCH=$(E2E_BRANCH) \
 	E2E_IP_FAMILY=$(E2E_IP_FAMILY) \
 	E2E_NETWORK_MODE=$(E2E_NETWORK_MODE) \
 	ginkgo $(GINKGO_PARALLEL_OPT) --randomize-all --always-emit-ginkgo-writer \
-		--focus=CNI:Kube-OVN ./test/e2e/ovn-eip/ovn-eip.test
+		--focus=CNI:Kube-OVN ./test/e2e/iptables-vpc-nat-gw/iptables-vpc-nat-gw.test
+
+.PHONY: ovn-vpc-nat-gw-conformance-e2e
+ovn-vpc-nat-gw-conformance-e2e:
+	ginkgo build $(E2E_BUILD_FLAGS) ./test/e2e/ovn-vpc-nat-gw
+	E2E_BRANCH=$(E2E_BRANCH) \
+	E2E_IP_FAMILY=$(E2E_IP_FAMILY) \
+	E2E_NETWORK_MODE=$(E2E_NETWORK_MODE) \
+	ginkgo $(GINKGO_PARALLEL_OPT) --randomize-all --always-emit-ginkgo-writer \
+		--focus=CNI:Kube-OVN ./test/e2e/ovn-vpc-nat-gw/ovn-vpc-nat-gw.test
```

Comment:
These test suites should be added and run in GitHub actions, which is defined in `.github/workflows/build-x86-image.yaml` and `.github/workflows/scheduled-e2e.yaml`.

Reply:
ok